### PR TITLE
TECH-683: Generate SSL certificate

### PR DESCRIPTION
**Pull request by mdulko**
ticket: https://govstack-global.atlassian.net/browse/TECH-683

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -41,6 +41,43 @@ services:
...
+      image: jrcs/letsencrypt-nginx-proxy-companion
+      restart: unless-stopped
+      volumes:
+        - /var/docker/nginx/html:/usr/share/nginx/html
+        - /var/docker/nginx/certs:/etc/nginx/certs
+        - /var/docker/nginx/vhost:/etc/nginx/vhost.d
+        - /var/run/docker.sock:/var/run/docker.sock
+        - /var/docker/nginx/acme:/etc/acme.sh
+
+    hello-world:
```
Comment (pgesek) [inline docker-compose.yml:72]: We do not need this hello world example, we should use our web container

File: docker-compose.yml [hunk 2]
```diff
@@ -4,12 +4,12 @@ services:
       build: .
       ports:
         - 80:3003
```
[the remark this answers is not in the document]
Comment (mdulko) [reply docker-compose.yml]: updated

File: docker-compose.yml [hunk 3]
```diff
@@ -41,6 +41,43 @@ services:
...
+        - /var/run/docker.sock:/var/run/docker.sock
+        - /var/docker/nginx/acme:/etc/acme.sh
+
+    hello-world:
+      container_name: hello-world
+      image: kornkitti/express-hello-world
+      expose:
+        - "8080"
+      environment:
+        VIRTUAL_HOST: test.programonaut.com
```
[the remark this answers is not in the document]
Comment (mdulko) [reply docker-compose.yml]: updated

File: deployment/run-prod.sh [hunk 4]
```diff
@@ -4,7 +4,8 @@ cd "$(dirname "$0")"
 
 # Do not use the override compose file, which is intended for development
 # We run in prod mode, but we still build from this guild repository
-docker-compose -f ../docker-compose.yml up --build -d --force-recreate 
+docker-compose -f ../docker-compose.prod.yml up --build -d --force-recreate
```
Comment (pgesek) [inline deployment/run-prod.sh]: We should use both files in one command, not use two commands

Comment (pgesek) [reply deployment/run-prod.sh]: I fixed this to save time

**Review by pgesek — APPROVED** [review]

**Review by pgesek — APPROVED** [review]

**Pull request merged**